packages/core/src/tracing/dynamicSamplingContext.ts

```diff
@@ -10,7 +10,7 @@ import {
 import type { DynamicSamplingContext } from '../types-hoist/envelope';
 import type { Span } from '../types-hoist/span';
 import { baggageHeaderToDynamicSamplingContext, dynamicSamplingContextToSentryBaggageHeader } from '../utils/baggage';
-import { extractOrgIdFromDsnHost } from '../utils/dsn';
+import { deriveOrgIdFromClient } from '../utils/dsn';
```

Comment:
l: why did we change this? IMHO `derive` is not easier to understand/grasp as `extract`...? 😅

Reply:
It's a different function :D
I extracted the logic of getting the org ID.

`extractOrgIdFromDsnHost` is still available: https://github.com/getsentry/sentry-javascript/pull/16313/files#diff-b6e0f71bb87e888ddb45cd78dcf2cb7efdf9c80b6d596c0224648b1f78bdcb29R148

Reply:
yeah, right, that makes sense of course - what I mean is that I think `derive` is a non-ideal term/prefix for the function name :D we do not use it anywhere in the code, rather we always use e.g. `extractXX` or `getXX` or something like this, so I'd rather use something along these lines for consistency - it is a different method of course :)